grails-app/domain/org/pih/warehouse/product/ProductPackage.groovy

```diff
@@ -54,7 +54,7 @@ class ProductPackage implements Comparable<ProductPackage>, Serializable {
         name(nullable: true)
         description(nullable: true)
         gtin(nullable: true)
-        uom(nullable: true)
```

Comment:
Was this expected to be removed? Is uom nullable or not?

Reply:
sorry, my bad.
I'll bring it back